src/Sylius/Bundle/ResourceBundle/spec/Controller/ConfigurationFactorySpec.php

```diff
@@ -20,17 +19,17 @@
  */
 class ConfigurationFactorySpec extends ObjectBehavior
 {
-    function let(ParametersParser $parametersParser)
+    public function let(ParametersParser $parametersParser)
```

Comment:
Can you revert changes in spec? We already talked many times about that they are not following PSRs and will never follow as they are generated.